project/profiles/capacity4more/modules/c4m/og/c4m_og_invite/c4m_og_invite.module

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file
+ * Code for og_invite and og_invite_people modules.
+ */
+
+/**
+ * Implements hook_menu_alter().
+ */
+function c4m_og_invite_menu_alter(&$items) {
+  $items['group/%/%/admin/people/invite']['access callback'] = 'c4m_og_invite_user_access_group';
+  $items['group/%/%/admin/people/invite-users']['access callback'] =
+  $items['group/%/%/admin/people/invite-users/%user']['access callback'] =
+  $items['group/%/%/admin/people/invite-users/autocomplete']['access callback'] = 'c4m_og_invite_access';
+}
+
+/**
+ * Invitation access callback.
+ */
+function c4m_og_invite_access($group_type, $gid) {
+  if (_og_invite_access($group_type, $gid)) {
+    return check_access_by_group_type(NULL, $group_type, $gid);
+  }
+  return FALSE;
+}
+
+/**
+ * Check if entity is a group, and user has permission - Access.
+ */
+function c4m_og_invite_user_access_group($perm, $group_type, $gid) {
+  if (og_invite_people_user_access_group($perm, $group_type, $gid)) {
+    return check_access_by_group_type($perm, $group_type, $gid);
+  }
+  return FALSE;
+}
+
+/**
+ * Return true if the user is allowed to invite and false if he isn't.
+ */
+function check_access_by_group_type($perm, $group_type, $gid) {
```

Comment:
@ybaras 
every function must contain its module name as a prefix, hence please rename the function so `c4m_og_invite_` will be the prefix of it.